### ISSUE TITLE
Named params are not properly handled for built-in functions

### ISSUE THREAD

**Issue by lihaoyi** [google/jsonnet#555]
It seems the names are ignored and params are treated as: when arguments are passed out of order they are just treated as positional arguments

This:

```javascript
local f(n, x) = std.pow(n, x);
[
  [std.pow(n=2, x=3), std.pow(x=3, n=2)],
  [f(n=2, x=3), f(x=3, n=2)]
]
```

Prints:

```json
[
  [
    8,
    9
  ],
  [
    8,
    8
  ]
]
```

When it should print:

```json
[
  [
    8,
    8
  ],
  [
    8,
    8
  ]
]
```

This is on v0.11.2

**Comment by sbarzowski** [comment]
Yes, this is a known issue, sorry for that. https://github.com/google/jsonnet/issues/385

The Go implementation doesn't have this problem: https://github.com/google/go-jsonnet. But the parameter names of builtins are not always the same...

This is something we must address, but for a time being it's best to simply avoid named arguments to builtin functions.
